### DISCUSSION TITLE
Spotify availability

### DISCUSSION THREAD

**Discussion by Panchajanya1999** [pear-devs/pear-desktop#575] [Q&A]
Thank you for your awesome app. 
Are there any such apps for Spotify with integrated download features and all?
Thanks.

**Comment by Araxeus** [comment]
https://github.com/mrpond/BlockTheSpot isn't exactly what you are asking for, but is great if you don't have premium

(if you end up using it, keep in mind that you need to run `BlockTheSpot.bat` every time Spotify updates)
